src/pages/lownKim/Login/Login.js

```diff
@@ -0,0 +1,110 @@
+// state에 값이 입력 될 때
+// 아이디값은 골뱅이가 들어있는지, 패스워드는 5글자 이상인지 판별해야함
+// 두 조건 모두 true라면
+// 버튼의 class명을 newButton으로 변경해야함
+// 그렇지 않으면 버튼 class명은 그대로 button
+
+// ex. ID - @ 포함 / PW - 5글자 이상
+// 만약 아이디 값 안에 골뱅이가 들어있고 패스워드가 5글자 이상이라면
+// if(this.state.email.includes("@") && this.state.password.length >= 5){
+// }
+
+// javascript.includes(java);  javascript에 java가 들어있나 확인하는 방법
+
+import React from 'react';
+import './Login.scss';
+
+class Loginlw extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      changeButton: 'button',
+    };
+  }
+  handleIdInput = e => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
+
+  handlepasswordInput = e => {
+    this.setState({
+      password: e.target.value,
+    });
+    // if (this.state.email.includes('@') && this.state.password.length >= 5) {
+    //   this.setState({ changeButton: 'newButton' });
+    // } else {
+    //   this.setState({ changeButton: 'button' });
+    // }
+    this.state.email.includes('@') && this.state.password.length >= 5
+      ? this.setState({ changeButton: 'newButton' })
+      : this.setState({ changeButton: 'button' });
+  };
+
+  render() {
+    //   let idValidation = false;
+    //   if (this.state.id.length > 5) {
+    //     idValidation = true;
+    //   }
+
+    console.log(this.state);
+
+    return (
+      <div className="wrapAll">
+        <section className="container">
+          <h1>
+            <span className="loginTitle">Westagram</span>
+          </h1>
```

Comment:
```suggestion
          <h1 className="loginTitle">Westagram</h1>
```
이런식으로 불필요한 depth를 지워주세요!